### PR TITLE
docs: Update our community docs page

### PR DESCRIPTION
**Pull request by pchaigno**
- Update the information on SIGs.
- Split the Slack channels in three categories and add a few new suggestions.
- Add a section on the Weekly Community Meeting.

### CODE REVIEW

**Review by qmonnet — APPROVED** [review]
Looks great, thanks!

File: Documentation/community.rst [hunk 1]
```diff
@@ -4,6 +4,67 @@
...
+Our Cilium & eBPF Slack is the main discussion space for the Cilium community.
+Click `here <https://cilium.herokuapp.com>`_ to request an invite. 
+
+Slack channels
+--------------
+
+==================== ====================================
+Name                 Purpose
+==================== ====================================
+#general             General user discussions & questions
```
Comment (pchaigno) [inline Documentation/community.rst:38]: I should also add the #hubble channel.

File: Documentation/community.rst [hunk 2]
```diff
@@ -4,6 +4,67 @@
...
+#kubernetes          Kubernetes-specific questions
+#networkpolicy       Questions on network policies
+#release             Release announcements only
+#sig-*               SIG-specific discussions (see below)
```
Comment (joestringer) [inline Documentation/community.rst]: Does this need to be here if it's also listed below for contributors?

Comment (pchaigno) [reply Documentation/community.rst]: I wasn't sure if we considered the SIG channels development channels or not. But we actually clarify that in a couple places already and we have #sig-hubble vs. #hubble and #sig-policy vs. #networkpolicy, so SIGs are clearly for development.

I removed this line.

File: Documentation/community.rst [hunk 3]
```diff
@@ -4,6 +4,67 @@
...
+- the current state of our CI: flakes being investigated and upcoming
+  changes
+- the development items for the next release
+- miscellaneous topics during the open session
+
+If you want to discuss something during the next meeting's open session,
+you can add it to `the meeting's Google doc
+<https://docs.google.com/document/d/1Y_4chDk4rznD6UgXPlPvn3Dc7l-ZutGajUv1eF0VDwQ/edit#>`_.
+
+Zoom link: https://zoom.us/j/596609673.
```
Comment (joestringer) [inline Documentation/community.rst]: Are we concerned at all about bots / trawlers / trolls finding these links and causing trouble?

If so, we could state that links to the meeting notes and Zoom room are available from the #development channel in Slack. At least then it requires going through a verification step first.

**Review by joestringer — COMMENTED** [review]
Just one remaining comment given potential for abuse. There's multiple ways to handle this, but I don't know if we're properly prepared to do so today. (eg docs could be completely rewritten, meeting video call could be zoombombed.)

File: Documentation/community.rst (hunk 3 above)
Comment (pchaigno) [reply Documentation/community.rst]: Sharing the link to the Google doc should be okay, no? I think we already share it sometimes in issues (at least I'm sure I saw it recently for the SIG-Datapath meeting notes).

Comment (joestringer) [reply Documentation/community.rst]: :+1: The google doc is easier to lock down if there's abuse, but we could also recreate the Zoom room if necessary.

**Review by joestringer — APPROVED** [review]

**Review by nebril — APPROVED** [review]

**Comment by pchaigno** [comment]
Marking for backport to ensure visitors always get the right information and to facilitate backports of subsequent changes to the same page.

**Pull request merged**